tavla/app/reset/[oob]/page.tsx

```diff
@@ -4,8 +4,8 @@ import { ResetForm } from './ResetForm'
 async function Reset(props: { params: Promise<{ oob: string }> }) {
     const params = await props.params
     return (
-        <div className="flex flex-col items-center eds-contrast">
-            <Heading3 className="mt-8 mr-2">Tilbakestill passord</Heading3>
+        <div className="flex sm:h-[50vh] flex-col items-center sm:justify-center mb-10">
```

Comment:
Vi kan sette egt `<main>` her i stedet for `<div>` (semantisk riktig), også legge til `container justify-center` på `className` og fjerne resten